.github/workflows/content-lint-markdown.yml

```
@@ -19,44 +25,37 @@ jobs:
 
       - name: Get changed content/data files
-        id: changed-files
-        uses: tj-actions/changed-files@635f118699dd888d737c15018cd30aff2e0274f8 # v44.0.1
+        id: changed_files
+        uses: ./.github/actions/get-changed-files
         with:
-          # No need to escape the file names because we make the output of
-          # tj-actions/changed-files be set as an environment variable. Not
-          # as a direct input to the line of bash that uses it.
-          safe_output: false
           files: |
```

actionlint (1.7.4)

input "files" is not defined in action "Get changed files" defined at "./.github/actions/get-changed-files". available inputs are "file_filter", "head", "output_file", "pr", "token"

(action)